microsoft-edge/toc.yml

```diff
@@ -1305,13 +1305,20 @@
           items:
             - name: Test upcoming APIs and features
               href: webview2/how-to/set-preview-channel.md
-    
+
+            - name: Prerelease testing
+              href: webview2/how-to/prerelease-testing.md
+
             - name: Use Playwright to automate and test in WebView2
               href: webview2/how-to/playwright.md
-    
+
             - name: Automate and test WebView2 apps with Microsoft Edge WebDriver
               href: webview2/how-to/webdriver.md
-    
+
+            - name: Self-hosting  # todo: move this TOC entry
```

Comment:
@victorhuangwq - move this toc entry to which existing bucket?
https://learn.microsoft.com/microsoft-edge/developer/

Reply:
I'm still working on updating the TOC buckets. What I will eventually do is to split out the current `Advanced Topics and Best Practices` bucket into `Advanced Topics` and `Best Practices` and then I will put both the pre-release testing and selfhosting under the `Best Practices`

Reply:
latest toc here in this PR seems ok, given this old TOC design.   @victorhuangwq 
```
# -----------------------------------------------------------------------------
        - name: Test and automation in WebView2 apps
          items:
            - name: Test upcoming APIs and features
              href: webview2/how-to/set-preview-channel.md

            - name: Use Playwright to automate and test in WebView2
              href: webview2/how-to/playwright.md

            - name: Automate and test WebView2 apps with Microsoft Edge WebDriver
              href: webview2/how-to/webdriver.md

        - name: Best Practices
          items:
            - name: Development best practices
              href: webview2/concepts/developer-guide.md
              displayName: Development best practices for WebView2 apps  # top-of-page title

            - name: Develop secure WebView2 apps
              href: webview2/concepts/security.md

            - name: Prerelease testing using preview channels
              href: webview2/how-to/prerelease-testing.md

            - name: Self-host by deploying preview channels
              href: webview2/how-to/self-hosting.md
```